### PR TITLE
Fixed urls and redirects

### LINKED ISSUE #106: Fix urls redirect flow
## Expected Behavior

- Homepage should be the first page you are landing on
- As Manager you should see New Task button
- As Employee you should not see New Task button

## Actual Behavior
Home page is not the first page, employee sees the New Task button

### PR DESCRIPTION
**Pull request by AvivLiberman**
Fixes #106 

## Changes you made
- Added app_user to the homepage and not auth_user
- fixed that only Manager sees New Task button
- Moved the navbar to a separate file
- added redirect to the Admin panel for admin users

### CODE REVIEW

**Review by avraham1992 — COMMENTED** [review]
You intend to create tasks only through the Admin django ?

File: templates/base.html [hunk 1]
```diff
@@ -1,62 +1,28 @@
...
-      {% block content %}{% endblock %}
-    </div>
-  </div>
-</main>
-    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js" integrity="sha384-ka7Sk0Gln4gmtz2MlQnikT1wXgYsOg+OMhuP+IlRH9sENBO0LRn5q+8nbTov4+1p" crossorigin="anonymous"></script>
-</body>
+    <title>Falcon</title>
+  </head>
+  <body>
+    {% include 'navbar.html' %} <div class="container"> {% block content %} {% endblock content %} <div>
```
Comment (orzionpour) [inline templates/base.html:20]:
```suggestion
    {% include 'navbar.html' %} 
    <div class="container"> 
        {% block content %} {% endblock content %}
    <div>
```

File: templates/homepage.html [hunk 2]
```diff
@@ -1,3 +1,3 @@
-{% extends "base.html" %}
-{% block content %}
+{% extends "base.html" %} {% block content %}
+<h1>HomePage</h1>
 {% endblock content %}
```
Comment (orzionpour) [inline templates/homepage.html:3]:
```suggestion
{% extends "base.html" %} 
{% block content %}
    <h1>HomePage</h1>
{% endblock content %}
```

File: templates/navbar.html [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+      </div>
+      <!-- Navbar Right Side -->
+      <div class="navbar-nav">
+        {% if user.user.is_authenticated %} {% if user.is_manager %}
+        <a class="nav-item nav-link" href="#">New Task</a>
+        {% endif %}
+        <a class="nav-item nav-link" href="{% url 'logout' %}">Logout</a>
+        {% else %}
+        <a class="nav-item nav-link" href="{% url 'login' %}">Login</a>
+        {% endif %}
```
Comment (orzionpour) [inline templates/navbar.html:27]:
```suggestion
        {% if user.user.is_authenticated %} 
            {% if user.is_manager %}
                <a class="nav-item nav-link" href="#">New Task</a>
            {% endif %}
                <a class="nav-item nav-link" href="{% url 'logout' %}">Logout</a>
            {% else %}
                <a class="nav-item nav-link" href="{% url 'login' %}">Login</a>
            {% endif %}
```

**Comment by AvivLiberman** [comment]
@avraham1992 No

**Review by orzionpour — CHANGES_REQUESTED** [review]
Just to make it a little more readable.

**Review by avraham1992 — COMMENTED** [review]
Why is it coming out of the base page (the nav bar) ?

**Review by avraham1992 — APPROVED** [review]

**Review by avraham1992 — APPROVED** [review]

**Review by orzionpour — DISMISSED** [review]

**Review by avraham1992 — DISMISSED** [review]

**Comment by natifridman** [comment]
I would also add a redirect from `/login` so if an authenticated user tries this he will be redirected and won't see the login form again.
Also, is there an option to create a redirect from the admin page too? So instead of seeing the default Django error, a regular user won't even be aware that there is an admin page. This is a security concern.

**Review by Asafh7 — DISMISSED** [review]

**Comment by mgold1234** [comment]
when I logged in with user that is not admin, I don't see the new task button and I suppose to see it. no?
image

the Falcon page is only with HomePage' - you need to fix it

**Comment by avraham1992** [comment]
> when I logged in with user that is not admin, I don't see the new task button and I suppose to see it. no? image
> 
> the Falcon page is only with HomePage' - you need to fix it

There should be no new task button after exiting only after entering and provided the user's role is manager...The only problem was that we had to give the page to our user so that he could use a role field and not an automatic one that not contains a field of function

**Review by aradz1997 — DISMISSED** [review]

**Comment by AvivLiberman** [comment]
> when I logged in with user that is not admin, I don't see the new task button and I suppose to see it. no? image
> 
> the Falcon page is only with HomePage' - you need to fix it

Only user with Role = Manager will see the New Task button

**Comment by AvivLiberman** [comment]
> I would also add a redirect from `/login` so if an authenticated user tries this he will be redirected and won't see the login form again. Also, is there any option to create a redirect from the admin page too? So instead of seeing the default Django error, a regular user won't even be aware that there is an admin page. This is a security concern.

Fixed the /login problem, the admin redirect should be in a different PR

**Review by orzionpour — APPROVED** [review]
Looks great :raised_hands: :raised_hands:

**Review by Asafh7 — APPROVED** [review]

**Comment by mgold1234** [comment]
please add a meaningful commit message

**Review by natifridman — APPROVED** [review]

**Review by kobihk — APPROVED** [review]

**Pull request merged**